rasterio/gdal.pxi

```diff
@@ -698,6 +698,54 @@ cdef extern from "gdal_alg.h" nogil:
             void * pTransformArg, double * padfGeoTransformOut, int * pnPixels,
             int * pnLines, double * padfExtent, int nOptions)
 
+cdef extern from "gdal_utils.h" nogil:
+    ctypedef struct GDALBuildVRTOptions:
```

Comment:
The C GDALBuildVRTOptions structure is an opaque one, which could evolve over time. Defining members as below should not be needed.

Reply:
Thanks @rouault :+1: